### PR TITLE
Fix some points to integrate

### PR DESCRIPTION
**Pull request by Konboi**
- AFAIR, we'll support Java 1.8 but we used Java 10 idiom `var`. Thus, fixed it
- Add token parse method to assemble request url.
- Add content-type header. Because we adopted gzip compressing.

### CODE REVIEW

File: src/main/java/io/jenkins/plugins/launchable/Ingester.java [hunk 1]
```diff
@@ -52,22 +50,25 @@ public boolean configure(StaplerRequest req, JSONObject json) throws FormExcepti
...
             // TODO: how do I obtain a properly configured HttpClient for the proxy setting in Jenkins?
             try (CloseableHttpClient httpClient = HttpClients.createDefault()) {
+                String endpoint = System.getenv("INSIGHT_UPLOAD_URL") ;
 
-                String endpoint = System.getenv("INSIGHT_UPLOAD_URL");
                 if (endpoint==null) {
                     endpoint = DEFAULT_UPLOAD_URL;
                 }
-                var hc = new HttpPost(endpoint);
+                HttpPost hc = new HttpPost(String.format("%s/intake/organizations/%s/workspaces/%s/events/jenkins", endpoint, orgWs.getOrganization(), orgWs.getWorkspace()));
```
Comment (Konboi) [inline src/main/java/io/jenkins/plugins/launchable/Ingester.java:63]: I'm not familiar with assembling URL part. Can I use `String.format`?

File: src/main/java/io/jenkins/plugins/launchable/Ingester.java [hunk 2]
```diff
@@ -52,22 +50,25 @@ public boolean configure(StaplerRequest req, JSONObject json) throws FormExcepti
...
 
-                var builder = MultipartEntityBuilder.create();
```
Comment (Konboi) [inline src/main/java/io/jenkins/plugins/launchable/Ingester.java:65]: Not to use Java 10 idiom

File: src/main/java/io/jenkins/plugins/launchable/Ingester.java [hunk 3]
```diff
@@ -52,22 +50,25 @@ public boolean configure(StaplerRequest req, JSONObject json) throws FormExcepti
...
+                MultipartEntityBuilder builder = MultipartEntityBuilder.create();
                 builder.addTextBody("metadata", properties.build().toString(), ContentType.APPLICATION_JSON);
                 builder.addBinaryBody("file", report, ContentType.APPLICATION_XML, "junitResult.xml");
 
                 hc.setEntity(new GzipCompressingEntity(builder.build()));
                 hc.addHeader("Authorization", "Bearer " + apiKey.getPlainText());
+                hc.setHeader(HttpHeaders.CONTENT_ENCODING, "gzip");
```
Comment (Konboi) [inline src/main/java/io/jenkins/plugins/launchable/Ingester.java:71]: We talked it this part directly, it seems server side can't handle compressed request correctly.

File: src/main/java/io/jenkins/plugins/launchable/Ingester.java (hunk 1 above)
Comment (kohsuke) [reply src/main/java/io/jenkins/plugins/launchable/Ingester.java:63]: Yes, this looks great

**Review by kohsuke — APPROVED** [review]

**Pull request merged**